<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1350</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NOLT</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
color: black;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelLogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>148</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="pixmap">
     <pixmap resource="../../images.qrc">:/Windows/Login/Images/loginBackground.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnColleagues">
    <property name="geometry">
     <rect>
      <x>902</x>
      <y>68</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Colleagues</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOffice">
    <property name="geometry">
     <rect>
      <x>1216</x>
      <y>68</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Office</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalary">
    <property name="geometry">
     <rect>
      <x>1094</x>
      <y>68</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Salary</string>
    </property>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1330</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1300</width>
      <height>492</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>160</red>
       <green>160</green>
       <blue>160</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QLabel" name="lblBackGroundPicture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1350</width>
      <height>680</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1350</width>
      <height>680</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Windows/Login/Images/appBackGround.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>lblBackGroundPicture</zorder>
   <zorder>labelLogo</zorder>
   <zorder>btnColleagues</zorder>
   <zorder>btnOffice</zorder>
   <zorder>btnSalary</zorder>
   <zorder>mdiArea</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
